UtopiaCity/Common/DbInitializer.cs

```diff
@@ -1,6 +1,5 @@
 ﻿using System.Collections.Generic;
 using UtopiaCity.Common.Initializers;
-using UtopiaCity.Common.Interfaces;
```

Comment:
remove using UtopiaCity.Common.Interfaces;

Reply:
and after removing that did you launch the project?

Reply:
Yes , I do . All working with out using UtopiaCity.Common.Interfaces;

Reply:
![image](https://user-images.githubusercontent.com/79470161/126749551-3f771327-5861-4e40-a223-03843737cdb7.png)